test/e2e/tests/dependencies.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/Azure/ARO-RP/test/e2e/tests/infra"
+)
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+
+	i, err := infra.NewInfrastructure()
```

Comment:
Dont need for a package withing a package. Set it as global object in E2E so destroy can be called in `AfterSuite`
```
i:= infra.New
i.Deploy()
i.Destory()
```